doc/design/error-handling.md

```diff
@@ -72,28 +68,27 @@ It is not supposed to be called by applications at any time.
 * When the reaction on an error is just logging, computation in other threads shall not be influenced.
 * If the error does not require termination, the error handler must return eventually.
 
-## Error Levels
+#### Error Levels
```

Comment:
I think this a subchapter of "Error Handling" and not of "Technical Requirements". If I'm right, please remove one ``#`` here and in the following error level subchapters.

Reply:
Fixed.